### PR TITLE
Correct Name on maven registry - Update pom.xml 

### PR DESCRIPTION
**Pull request by sahithikkss-zz**
QnA Maker java sdk is being shown as Vision sdk on https://mvnrepository.com/search?q=qnamaker

image

### CODE REVIEW

File: sdk/cognitiveservices/ms-azure-cs-qnamaker/pom.xml [hunk 1]
```diff
@@ -15,11 +15,11 @@
...
     <groupId>com.microsoft.azure.cognitiveservices</groupId>
-    <version>1.0.0-beta</version> <!-- {x-version-update;com.microsoft.azure.cognitiveservices:azure-cognitiveservices-qnamaker;current} -->
+    <version>1.0.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure.cognitiveservices:azure-cognitiveservices-qnamaker;current} -->
     <packaging>jar</packaging>
-    <name>Microsoft Azure SDK for Cognitive Service Computer Vision</name>
-    <description>This package contains Microsoft Cognitive Service Computer Vision SDK.</description>
-    <url>https://github.com/Azure/azure-sdk-for-java/cognitiveservices/data-plane/vision/qnamaker</url>
+    <name>Microsoft Azure SDK for Cognitive Service QnA Maker</name>
+    <description>This package contains Microsoft Cognitive Service QnA Maker SDK.</description>
+    <url> https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/cognitiveservices/ms-azure-cs-qnamaker</url>
```
Comment (conniey) [inline sdk/cognitiveservices/ms-azure-cs-qnamaker/pom.xml:22]: nit: unneeded space. after `<url>`

Comment (sahithikkss) [reply sdk/cognitiveservices/ms-azure-cs-qnamaker/pom.xml]: good catch. removed the space. Thanks.

**Review by joshfree — APPROVED** [review]

**Comment by sahithikkss** [comment]
Mistakenly closed. So, reopened.
Can you please merge the changes as well.
I do not see an option to do that.
@bingisbestest, @joshfree

**Comment by sahithikkss** [comment]
@joshfree , could you please merge this as well as I am not authorized to do so.

**Comment by sahithikkss** [comment]
> @joshfree , could you please merge this as well as I am not authorized to do so.

can anyone please help with failing checks and merge?

**Comment by joshfree** [comment]
per offline discussion @jianghaolu will help get this change merged and the SDK package re-released with this change.

Thank you for your contribution @sahithikkss

**Review by jianghaolu — APPROVED** [review]

**Comment by jianghaolu** [comment]
Thanks for the contribution - please also update the entry at https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_data.txt#L26 to unblock CI.

**Comment by sahithikkss** [comment]
> Thanks for the contribution - please also update the entry at https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_data.txt#L26 to unblock CI.

@jianghaolu Changed version as suggested (to 1.0.0-beta.1). Please merge as applicable.

**Pull request merged**